lib/formatters/verboseFormatter.js

```diff
@@ -83,3 +84,30 @@ function groupBy(array, keyFn) {
 
 	return result;
 }
+
+/**
+ * @param {string | undefined} source
+ * @returns {string}
+ */
+function fileLink(source) {
+	if (!source || source.startsWith('<')) {
+		return `${source}`;
+	}
+
+	return terminalLink(source, `file://${source}`, { fallback: false });
+}
+
+/**
+ * @param {string} rule
+ * @returns {string}
+ */
+function ruleLink(rule) {
+	// Plugin rules are unsupported.
+	if (rule.includes('/')) {
+		return rule;
+	}
```

Comment:
[note] It is a bit disappointing not to support plugin rules.
We may need to add a rule’s metadata (rule doc URL etc.) in the future. 🤔 
(but out of the scope of this PR)

See https://eslint.org/docs/developer-guide/working-with-rules#rule-basics

Reply:
I've opened a new issue #5842 to discuss rule metadata.

Reply:
We could use `meta.url` if #5845 would be merged.